packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -9,48 +9,63 @@ import { logViewEvent } from '../../../lib/metrics';
 import Ready from '../../../components/Ready';
 import { MozServices } from '../../../lib/types';
 import { REACT_ENTRYPOINT } from '../../../constants';
+import AppLayout from '../../../components/AppLayout';
+import { useFtlMsgResolver } from '../../../models';
 
 type ResetPasswordWithRecoveryKeyVerifiedProps = {
   serviceName?: MozServices;
+  // Verify if relier is Sync with the useRelier hook that will be implemented in FXA-6437
+  // instead of using a prop
+  isSync?: boolean;
 };
 
 export const viewName = 'reset-password-with-recovery-key-verified';
 
+// This page does not currently perform a check to verify that the password has indeed been reset.
+// It is possible to directly hit this route when no password reset has been initiated/completed,
+// even when the user is not signed in.
+
```

Comment:
This is the current behaviour in content-server, but I feel we should change it and add a check so this page doesn't load unless a password reset has succeeded. Perhaps we can pass in state from `AccountRecoveryResetPassword` @dschom? I'm not sure what might be the best way to implement this check.

Reply:
I tend to agree. This feels like follow up territory to me though. Maybe file a separate issue? 

Off the top of my head, I am not sure the best way to do this. There's a chance we don't explicitly record this state anywhere at the moment, which means we'd need to decide how to persist it between views (i.e. on the client side somehow or perhaps in the database).

Reply:
Thanks Dan, this did feel a bit out of scope for this ticket, so I've filed a new issue: https://mozilla-hub.atlassian.net/browse/FXA-6805